integration/tests/main/03-create-private-package-repository.spec.js

```diff
@@ -0,0 +1,130 @@
+const { test, expect } = require("@playwright/test");
```

Comment:
Just noticed we're missing the SPDX headers with which the old tests were already updated.

Reply:
You are right, will add them.